storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1207,6 +1207,14 @@ bool Redo_Log_Data_Manager::stop_at(lsn_t lsn, lsn_t checkpoint_lsn) {
     return (false);
   }
 
+#ifndef DBUG_OFF
+  if (scanned_lsn > stop_lsn) {
+    msg("xtrabackup: warning: the last copied LSN " LSN_PF
+        " is larger than last lsn from log_status" LSN_PF ".\n",
+        last_checkpoint_lsn, scanned_lsn);
+  }
+#endif /* !DBUG_OFF */
+
```

Comment:
ut_ad(scanned_lsn <= stop_lsn);

Reply:
I want this assertion after the redo copy is over. Or after the redo thread/file is closed.